compiler/ilgen/IlBuilder.hpp

```diff
@@ -27,7 +27,9 @@
 
 #include <fstream>
 #include <stdarg.h>
-#include <string.h>
+#include <string.h>   // TODO: Remove
```

Comment:
I was planning to remove it when I got rid of all `char *` in this file. However, I don't see any `string.h` functions being used anyway - so I'll remove it as part of this commit.